finality-aleph/src/sync/mock/backend.rs

```diff
@@ -30,6 +30,7 @@ struct BackendStorage {
 pub struct Backend {
     inner: Arc<Mutex<BackendStorage>>,
     notification_sender: UnboundedSender<MockNotification>,
+    headers_are_correct: Arc<Mutex<bool>>,
```

Comment:
nit: rename to `accepting_headers` or smth like this

Reply:
This is such a weird way of doing this, cannot you add something to headers instead? Even an `is_correct: bool` in `MockHeader` would work, wouldn't it?

Reply:
+1 to timrl[eph] suggestion

Reply:
rewritten

Reply:
Cool, @timorleph leaving this conversation up to you to resolve